specification/servicebus/resource-manager/readme.md

```diff
@@ -238,5 +257,36 @@ See configuration in [readme.go.md](./readme.go.md)
 
 See configuration in [readme.java.md](./readme.java.md)
 
+## Suppression
+
+``` yaml
+directive:
+  - suppress: ConsistentPatchProperties
+    from: namespace-preview.json
+    reason: Not a mandatory check
+
+  - suppress: ResourceNameRestriction
+    from: namespace-preview.json
+    reason: Not a mandatory check
+  
+  - suppress: ResourceNameRestriction
+    from: AuthorizationRules.json
+    reason: Not a mandatory check
+  - suppress: PathResourceTypeNameCamelCase
+    from: AuthorizationRules.json
+    reason: Not a mandatory check
+  - suppress: RequestSchemaForTrackedResourcesMustHaveTags
+    from: AuthorizationRules.json
+    reason: Not a mandatory check
+  - suppress: PutResponseCodes
+    from: AuthorizationRules.json
+    reason: Not a mandatory check
+  - suppress: TrackedResourcePatchOperation
+    from: AuthorizationRules.json
+    reason: Not a mandatory check
+  - suppress: SystemDataDefinitionsCommonTypes
+    from: AuthorizationRules.json
+    reason: Not a mandatory check
```

Comment:
These are all mandatory checks. Please provide proper justifications.

Reply:
Hey @rkmanda , I have given justification for suppression 
can you pls, approve now.